analyzer/pkg/rpm/rpm.go

```diff
@@ -30,6 +30,19 @@ var requiredFiles = []string{
 }
 var errUnexpectedNameFormat = xerrors.New("unexpected name format")
 
+var osVendors = []string{
+	"Amazon Linux",     // Amazon Linux 1
+	"Amazon.com",       // Amazon Linux 2
+	"CentOS",           // CentOS
+	"Fedora Project",   // Fedora
+	"Oracle America",   // Oracle Linux
+	"Red Hat, Inc.",    // Red Hat
+	"AlmaLinux",        // AlmaLinux
+	"CloudLinux",       // AlmaLinux
+	"VMware",           // Photon
```

Comment:
I found `VMware, Inc.` in `photon:3.0` In which version did you find it?

Reply:
That was my fault. `VMware, Inc.` is the right vendor. Thanks for finding this. I changed this.